types/kafka/kafka-retry-consumer.d.ts

```diff
@@ -0,0 +1,6 @@
+export type KafkaRetryBindOptions = {
```

Comment:
Extend from `BindOptions`

Reply:
Whatever the valid combinations are they should be in the type. Right now it's possible to pass `{}`

Reply:
It's possible but nothing will happen so best if we prevent it with the type system, no? Because I don't think you should be able to pass no callback to a bind. I think not passing `onSuccess`, `onError` and `retries` is fine, but `callback` should always be there (since there is no default and a bind call with no action seems like a bug)